app/Content/routes/content-routes.php

```diff
@@ -1,12 +1,30 @@
 <?php
 
 use App\Content\Http\Controllers\Auth\AuthController;
+use App\Content\Http\Controllers\BlogPostController;
+use App\Content\Http\Controllers\BlogPostTagsController;
 
 Route::group(['prefix' => '/content'], function () {
     Route::post('/login', [AuthController::class, 'login'])->middleware(['throttle:login']);
 
     Route::middleware(['auth:sanctum', 'verified'])->group(function () {
         Route::post('/logout', [AuthController::class, 'logout']);
+
+        Route::group(['prefix' => '/blog-posts'], function () {
+            Route::get('/', [BlogPostController::class, 'index']);
+            Route::post('/', [BlogPostController::class, 'store']);
+            Route::get('/{blogPost}', [BlogPostController::class, 'show']);
+            Route::patch('/{blogPost}', [BlogPostController::class, 'update']);
+            Route::delete('/{blogPost}', [BlogPostController::class, 'destroy']);
+            Route::get('/changeStatus/{blogPost}', [BlogPostController::class, 'changeStatus']);
```

Comment:
`{blogPost}/statuses` makes more sense here, also should it be a GET or a PATCH request? Also instead of changeStatus, you could put this code into a new controller called `BlogPostStatusController` and add a method called `update`.